modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -0,0 +1,89 @@
+= Build SOAP API Proxies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API Manager 2.x enables you to use SOAP API proxies for your SOAP-based web services. The SOAP proxy requires a WSDL definition that describes the backend service. The  WSDL definition enables the API proxy to perform validations on incoming requests and correctly routes outbound requests to the corresponding API endpoint. 
+
+Based on your configuration, validations are performed either against the SOAP message schema and the envelope, or only the envelope. By default, SOAP message schema validations are disabled.
+
+== How schemas are Validated on SOAP Proxies
+
+When you enable validations for an API, the SOAP action in the inbound request is matched against the same-named operation defined in the WSDL definition. After the matching operation is obtained, the proxy validates whether the namespace URI of the element and the name associated with the operation matches the same details in the incoming request. 
+
+If the validation fails, an error message informing the expected namespace URI and element name is generated. If the SOAP action in the inbound request is empty, the application attempts to match the namespace URI and element name of the incoming request element to all the operations defined in the WSDL definition. 
```

Comment:
This because the word "definition" after WSDL might be a little redundant since WSDL stands for Web Services Description Language: https://www.w3.org/TR/wsdl.html

Reply:
Done! Thanks!